docs/source/usage.rst

```diff
@@ -120,22 +93,3 @@ be used by docker directly.
   .. code-block:: bash
 
      jupyter-repo2docker --no-build --debug https://github.com/jakevdp/PythonDataScienceHandbook
-
-Setting environment variables
```

Comment:
I'd keep both of these questions.

Reply:
Can someone explain the use case of environment variables? I took this out because I didn't understand the documentation and decided to move it to just the FAQ